tendrl/ceph_integration/objects/tendrl_context/__init__.py

```diff
@@ -20,8 +20,25 @@ def __init__(self, integration_id=None, fsid=None, *args, **kwargs):
         self.integration_id = integration_id or self._get_local_integration_id()
         self.fsid = fsid or self._get_local_fsid()
         self.sds_name, self.sds_version = self._get_sds_details()
+        self.integration_name = self._get_integration_name()
         self._etcd_cls = _TendrlContextEtcd
 
+    def _get_integration_name(self):
+        cluster_name = ""
+        # TODO(team) This file is valid for CentOS and RHEL. Needs to
+        # be handled while ubuntu support is needed.
+        ceph_cfg_file = "/etc/sysconfig/ceph"
+        if not os.path.exists(ceph_cfg_file):
+            LOG.warning("config file: %s not found" % ceph_cfg_file)
+            return cluster_name
```

Comment:
@nthomas-redhat what is the default ceph cluster name if none is supplied during cluster creation? We should return that default name here